examples/src/main/java/tachyon/examples/BasicRawTableOperations.java

```diff
@@ -126,12 +131,13 @@ private void write(TachyonFS tachyonClient) throws IOException {
   }
 
   public static void main(String[] args) throws IllegalArgumentException {
-    if (args.length != 3) {
+    if (args.length != 4) {
       System.out.println("java -cp " + Constants.TACHYON_JAR
-          + " tachyon.examples.BasicRawTableOperations <TachyonMasterAddress> <FilePath>");
+          + " tachyon.examples.BasicRawTableOperations <TachyonMasterAddress> <FilePath>"
```

Comment:
`BasicRawTableOperations.class.getName() + " <master address> <file path> <tachyon storage type for writes (STORE | NO_STORE)> <under storage type for writes (SYNC_PERSIST | NO_PERSIST)>"`